tools/accuracy_checker/configs/README.md

```diff
@@ -0,0 +1,34 @@
+# How to use configuration files
+
+## Structure
+
+Configuration file declares validation process. Every model has to have entry in `models` list. Each entry has to contain distinct `name`, `launchers` and `datasets` sections.
```

Comment:
please specify how to run composite model, because they have different config structure